main.tf

```diff
@@ -186,6 +186,23 @@ resource "aws_s3_bucket" "default" {
             replica_kms_key_id = try(rules.value.destination.replica_kms_key_id, null)
             account_id         = try(rules.value.destination.account_id, null)
 
+            # https://docs.aws.amazon.com/AmazonS3/latest/userguide/replication-walkthrough-5.html
+            dynamic "metrics" {
+              for_each = try(rules.value.destination.metrics.status, "") == "Enabled" ? [1] : []
+
+              content {
+                status = "Enabled"
+              }
```

Comment:
See other comment with the same suggestion.

```suggestion
              content {
                status = "Enabled"
                # we are not hardcoding this value — rather, this is the only value accepted by the provider, either implicitly or explicitly.
                minutes = 15
              }
```

Reply:
minutes are ignored here

correct syntax is

metrics {
  event_threshold {
    minutes = 15
  }
  status = "Enabled"
}


but it's not supported by the latest aws provider, I don't know why

Reply:
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket_replication_configuration

Reply:
I don't think that's the correct syntax

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket#using-replication-configuration

<img width="699" alt="CleanShot 2022-01-31 at 14 56 53@2x" src="https://user-images.githubusercontent.com/10080107/151797503-f191a0c8-006f-4bda-9be4-66a2efa8e5f7.png">

Reply:
https://github.com/hashicorp/terraform-provider-aws/blob/d58a414f2a0647b2b62fe6e66744196cae21d358/internal/service/s3/bucket.go#L459-L500